Projekty/Projekt1/Gruszkowska_Klaudia_Jamrozy_Bartosz/KM2.ipynb

```diff
@@ -0,0 +1,3476 @@
+{
```

Comment:
<p>bardzo podobne linijki kodu powtarzają się wielokrotnie - należałoby to obudować w funkcję.</p><p>Zabrakło zbiorczego podsumowania jakości wszystkich modeli - mogłaby to być np. tabelka.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>